pydantic/fields.py

```diff
@@ -12,6 +12,7 @@
 
 import annotated_types
 import typing_extensions
+from typing_extensions import Unpack
```

Comment:
Doing this so the docs look cute:
<img width="693" alt="image" src="https://github.com/pydantic/pydantic/assets/7353520/45064c41-cf6d-4b26-8a8b-676e901148b4">